examples/grpc-server-with-prometheus/server/server.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+
+	"github.com/soheilhy/cmux"
```

Comment:
Can you please avoid the unnecessary dependency and use standard go packages?

Reply:
yea, especially for demo - we want to have it as clear as possible, and someone can be not familiar with `cmux`

Reply:
I will remove it